pgmuvi/lightcurve.py

```diff
@@ -328,6 +328,41 @@ def __init__(self, xdata, ydata, yerr=None,
         self.__FITTED_MAP = False
         self.__FITTED_MCMC = False
 
+    @classmethod
+    def from_table(cls, VOTable, **kwargs):
+        """Instantiate a Lightcurve object with
+        data read in from a VOTable.
+
+        Parameters
+        ----------
+        VOTable : str
+            Name (with extension) of the VOTable containing input data
+        kwargs:
+            Arguments to be passed to the Lightcurve call
+        
+        Returns
+        ----------
+        Lightcurve object
+        """
+        from astropy.table import Table
+        data = Table.read(VOTable, format='votable')
```

Comment:
Does it have to be hardcoded to a VOTable? Couldn't it be any other type of astropy table?